launcher/src/backend/ethereum-services/LighthouseBeaconService.js

```diff
@@ -22,9 +22,13 @@ export class LighthouseBeaconService extends NodeService {
     // eth1 nodes
     const eth1Nodes = executionClients
       .map((client) => {
-        const elJWTDir = client.volumes.find((vol) => vol.servicePath === "/engine.jwt").destinationPath;
+        const elJWTDir = client.volumes.find(
+          (vol) => vol.servicePath === "/engine.jwt" || vol.destinationPath.includes("/engine.jwt")
+        ).destinationPath;
         volumes.push(new ServiceVolume(elJWTDir, JWTDir));
-        return client.buildExecutionClientEngineRPCHttpEndpointUrl();
+        return client.service?.includes("ExternalExecutionService")
+          ? client.buildExecutionClientEngineRPCHttpEndpointUrl(client.env.link)
```

Comment:
see above comment. make the check in the `External` - Classes

Reply:
this goes for all other occurrences as well

Reply:
the problem is not only the extra code, if `client.env.link` is `undefined`, the string `undefined` ends up in the config

Reply:
see above comment